lib/howl/ui/style.moon

```diff
@@ -234,6 +234,19 @@ set_for_theme = (theme) ->
   for name in pairs styles
     define_extended name if is_extended name
 
+ -- set status styles
```

Comment:
This is an attempt to remove the duplicated definition for window.status.(info|warning|error) and editor.styles.(info|warning|error). It just copies the former into editor.status.status-<level>.

Should I remove this?